src/server/game/Entities/Player/Player.cpp

```diff
@@ -26918,4 +26947,3 @@ void Player::SendSupercededSpell(uint32 oldSpell, uint32 newSpell)
     data << uint32(oldSpell) << uint32(newSpell);
     GetSession()->SendPacket(&data);
 }
-
```

Comment:
Just found out why I made this change. When I clone the source from the TC branch, the file appears to have 2 blank lines at the end when I open Player.cpp in Notepad++ - hence the removal of a blank line at the end.